holidays/countries/pakistan.py

```
@@ -89,6 +100,15 @@ def _populate_public_holidays(self):
         self._add_ashura_eve(name)
         self._add_ashura_day(name)
 
+    self._populate_subdiv_holidays()
```

Ruff (0.8.2)

Undefined name `self`

(F821)